### DISCUSSION TITLE
Elfin EW11 was working now not working!

### DISCUSSION THREAD

**Discussion by ibby24** [nathanmarlor/foxess_modbus#693] [General]
I'm getting an error saying that the entitys were offline. 
So I rebooted HA. It didn't fix it 
I rebooted the router and the elfin it didn't work. 
I removed the integration and reinstalled it it didn't work. 
5 hours before that everything was working!
I've double checked all the settings

I just get an error saying
Error connecting to your adapter or inverter ensure the adapter is properly configured and correctly wired to the inverter. 
Any ideas anyone?

**Comment by FozzieUK** [comment]
If it was all working and no changes were made, I would suggest in the first instance check that the IP address you allocated to the Elfin EW11 hasn't changed - it's possible it's DHCP lease expired and a new IP address has been allocated to it by your router.
